plugins/image-prioritizer/class-image-prioritizer-background-image-styled-tag-visitor.php

```diff
@@ -39,9 +39,9 @@ public function __invoke( OD_HTML_Tag_Walker $walker ): bool {
 		if (
 			is_string( $style )
 			&&
-			0 < (int) preg_match( '/background(-image)?\s*:[^;]*?url\(\s*[\'"]?\s*(?<background_image>.+?)\s*[\'"]?\s*\)/', $style, $matches )
+			1 === preg_match( '/background(?:-image)?\s*:[^;]*?url\(\s*[\'"]?\s*(?<background_image>.+?)\s*[\'"]?\s*\)/', $style, $matches ) // Not compatible with strict rules. See <https://github.com/phpstan/phpstan/issues/11262>.
```

Comment:
Bah. I should have removed the comment as it is now irrelevant after 75cd653dd3853f9038f5e26abae3f96256ddb9a3 per https://github.com/phpstan/phpstan/issues/11262#issuecomment-2200728974. I'll do so as part of another PR.